assets/src/components/routeLadder.tsx

```diff
@@ -66,14 +70,52 @@ export const Header = ({
         </Tippy>
       )}
       <div className="c-route-ladder__close-button-container">
-        <CloseButton closeButtonType="l_darker" onClick={onClose} />
+        <OldCloseButton closeButtonType="l_darker" onClick={onClose} />
       </div>
 
       <div className="c-route-ladder__route-name">{routeName}</div>
     </div>
   )
 }
 
+// TODO: delete old header after roll-out
+export const NewHeader = ({
+  routeName,
+  onClose,
+  hasAlert,
+}: {
+  routeName: string
+  onClose: () => void
+  hasAlert: boolean
+}) => {
+  return (
+    <Card className="c-new-route-ladder__header">
+      <Card.Body>
+        <div className="c-new-route-ladder__fake-dropdown" />
```

Comment:
Non-blocking: Did we ever determine what the UX here should be if the user isn't allowed to add/edit detours? I assume that this "fake" dropdown shouldn't be present. 

This is a non-blocking comment because this isn't going live yet, but I'm _pretty_ sure that this is something we'll need to address before we do go live.

Reply:
No need! I don't think this needs to be addressed in THIS PR - I just wanted to make sure it wasn't fully dropped.